config/settings.yml

```diff
@@ -12,6 +12,13 @@ financial:
     csv: true
     xml: true
 
+converters:
+  default:
+    product_to_journal_rows: Converters::ProductToJournalRows
+    order_detail_to_journal_rows: Converters::OrderDetailToJournalRows
+  split_accounts/split_account:
+    order_detail_to_journal_rows: SplitAccounts::Converters::OrderDetailToJournalRows
```

Comment:
What is the advantage of having all of this configurable? Could there be a method on `Account#order_detail_journal_row_converter` method that returns the default `Converters::OrderDetailToJournalRows`. Then `SplitAccount` could override that.